src/core/modules/query.js

```diff
@@ -77,6 +77,29 @@ const _extend = {
     }
 };
 
+/**
+ * @function getRequestUrl
+ * @description Returns host url based on this.type
+ * @param  {Object} this `this` variable from Query class
+ * @return {string} returns the url that will be used to make API calls
+ */
+function getRequestUrl(type, config, content_type_uid, baseURL) {
```

Comment:
Please check the coverage report to make sure we do not miss tests @abhinav-from-contentstack

Reply:
Hi @ishaileshmishra , I've added a test to cover this method completely.